packages/plugin-node-surrounding-code/surrounding-code.js

```diff
@@ -1,5 +1,8 @@
 const SURROUNDING_LINES = 3
-const { readFile } = require('fs')
+const { createReadStream } = require('fs')
+const { Writable } = require('stream')
+const pump = require('pump')
+const byline = require('byline')
```

Comment:
this module is used to buffer the file read stream up to each line break and emit chunks which are single line of code